1/src/com/company/SetOperationsImplements.java

```diff
@@ -0,0 +1,43 @@
+package com.company;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Created by InnaBakum on 09.11.2015.
+ */
+public class SetOperationsImplements  implements SetOperations{
```

Comment:
Don't forget about code convention, should be space between SetOperations and  {